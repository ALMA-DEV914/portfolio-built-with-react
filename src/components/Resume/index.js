import React from "react";

function Resume() {
  return (
    <main>
    <section>
      <h1>My Resume</h1>
      <a href={require("../../assets/Resume.pdf")} download>
        Download my resume
      </a>
      <div>
          <h2>Front End Proficiences</h2>
          <ul>
              <li>HTML</li>
              <li>CSS</li>
              <li>Bootstrap</li>
              <li>JAVASCRIPT</li>
              <li>JQUERY</li>
              <li>Responsive Design</li>
              <li>REACT</li>
          </ul>
      </div>
      <div>
          <h2> Back End Proficiences</h2>
          <ul>
              <li>API's</li>
              <li>NODE.JS</li>
              <li>MySQL/NoSQL Sequelize</li>
              <li>MONGODB database</li>
              <li></li>
          </ul>
      </div>
    </section>
    </main>
  );
}

export default Resume;