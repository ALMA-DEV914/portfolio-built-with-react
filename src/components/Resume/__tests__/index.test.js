import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Resume from '..';

afterEach(cleanup);

describe('About component', () => {
    //renders About test
    it('renders', () => {
        render(<Resume />);
    });
    it('matches snapshot DOM node structure', () => {
        const {asFragment} = render(<Resume />);
        expect(asFragment()).toMatchSnapshot();
    });
})
