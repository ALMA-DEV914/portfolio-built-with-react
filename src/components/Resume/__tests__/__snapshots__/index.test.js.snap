// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <main>
    <section
      class="resume"
    >
      <h1>
        My Resume
      </h1>
      <br />
      <a
        download=""
        href="Resume.pdf"
        id="pdf"
      >
        Download my resume
      </a>
      <article>
        <div>
          <h2>
            Front End Proficiences
          </h2>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              Bootstrap
            </li>
            <li>
              JAVASCRIPT
            </li>
            <li>
              JQUERY
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              REACT
            </li>
          </ul>
        </div>
        <div>
          <h2>
             Back End Proficiences
          </h2>
          <ul>
            <li>
              API's
            </li>
            <li>
              NODE.JS
            </li>
            <li>
              MySQL/NoSQL Sequelize
            </li>
            <li>
              MONGODB database
            </li>
            <li />
          </ul>
        </div>
      </article>
    </section>
  </main>
</DocumentFragment>
`;
